core/src/empty/empty.tsx

```diff
@@ -0,0 +1,30 @@
+import s from "./empty.module.css";
+import { createElement } from "react";
+import { Button } from "../button/button";
+import cloudImg from "./empty1.jpg";
+
+interface EmptyProps {
+    active: boolean,
+    errorMsg: string,
+    onClickBtn?: React.MouseEventHandler
```

Comment:
The naming should be more specific. Instead of "onClickBtn", I suggest "actionHandler" or just "action" is enough.

The point here is that we'd like an action. Maybe it's "go to home" or "try reload", we dont know